src/bootloaders/extlinux.c

```diff
@@ -238,22 +238,22 @@ static bool syslinux_install(const BootManager *manager)
                 return false;
         }
 
-        syslinux_path = string_printf("%s/usr/share/syslinux/gptmbr.bin", prefix);
```

Comment:
This is broken, extlinux isn't a distro thing. The gptmbr.bin is /usr/share/syslinux/gptmbr.bin as the package is syslinux.

Reply:
Obsolete - we embed the MBR's now in the code to avoid relying on different paths in different distro's. See 4bd4332 .

Reply:
Ah yes, I saw that though I have concerns about that in and of itself but it does address this comment.